kolibri/core/views.py

```diff
@@ -3,10 +3,14 @@
 from django.core.urlresolvers import translate_url
 from django.utils.http import is_safe_url
 from django.utils.translation import LANGUAGE_SESSION_KEY, check_for_language
+from django.views.decorators.csrf import csrf_exempt, ensure_csrf_cookie
 from django.views.i18n import LANGUAGE_QUERY_PARAMETER
 
 
+
```

Comment:
One less blank line here - caused a linting error: `kolibri/core/views.py:11:1: E303 too many blank lines (3)`

Reply:
Okay, i'll remove that.